arrow-libs/optics/arrow-optics/src/main/kotlin/arrow/optics/std/validated.kt

```diff
@@ -1,17 +0,0 @@
-package arrow.optics
-
-import arrow.core.Either
-import arrow.core.Validated
-
-/**
- * [PIso] that defines equality between [Validated] and [Either]
- */
-fun <A1, A2, B1, B2> Validated.Companion.toPEither(): PIso<Validated<A1, B1>, Validated<A2, B2>, Either<A1, B1>, Either<A2, B2>> = PIso(
```

Comment:
These are present on `0.11.0` and cannot be removed. They need to be deprecated in favor of `Iso.pEither()` and `Iso.either()`.

Reply:
Idem, should've been `validatedToPEither` and same for the deprecation